src/Game.tsx

```
@@ -50,47 +52,50 @@ function Game() {
 
   const currentTurn = xIsNext ? "X's Turn" : "O's Turn";
 
+  const currentTurn = xIsNext ? "X's Turn" : "O's Turn";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'currentTurn'. Consider to delete it or rename it.

'currentTurn' is defined here:


(lint/suspicious/noRedeclare)